mass_mailing_sending_queue/models/mail_mass_mailing.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning as UserError
+
+
+class MailMassMailingSending(models.Model):
+    _inherit = 'mail.mass_mailing'
+
+    state = fields.Selection(selection_add=[
+        ('sending', "Sending"),
+    ])
+    pending = fields.Integer(string="Pending", compute='_compute_pending')
+
+    @api.multi
+    def send_mail(self):
+        # Override 'Send to All' button completly.
+        # NOTE: Please, notice that this breaks inheritance chain
```

Comment:
Another approach would be to have a separate function where you do the queuing. Then you could adapt the UI in a way that it's either the user's choice which button to click or maybe steered by group membership who does what.
Then you could also use this function in your cronjob and profit from whatever other modules do here